src/libs/thorvg/thorvg_capi.h

```diff
@@ -433,7 +435,7 @@ typedef enum {
 *
 * \return A new Tvg_Canvas object.
 */
-TVG_API Tvg_Canvas* tvg_swcanvas_create(void);
+TVG_API Tvg_Canvas* tvg_swcanvas_create();
```

Comment:
Please keep the `void` parameters in this file. I'm sending a PR to ThorVG about it.

Reply:
Already done in https://github.com/thorvg/thorvg/pull/2316

Reply:
Great, I've closed mine: https://github.com/thorvg/thorvg/pull/2323

Reply:
done